post-processing/streamlit_post_processing.py

```diff
@@ -169,6 +177,23 @@ def axis_select(label: str, axis: dict):
 
     # units select
     units_select(label, axis)
+
+    # FIXME: add ability to use a custom value for only one of min or max
+    disable_user_range = plot_type != 'line'
+    use_default_ranges = st.checkbox("default axis range",
+                                     axis.get("range").get("use_default") if not disable_user_range else True,
```

Comment:
Minor nitpick, but is this `use_default` bool for the axis ranges particularly useful anywhere else? It's technically information we can derive (i.e. `use_default = axis.get("range").get("min") is None and axis.get("range").get("max") is None`) and might feel like a slightly redundant field for anyone creating a plot config the manual way.